fv/wireguard_test.go

```diff
@@ -994,6 +985,30 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ WireGuard-Supported 3-node
 		}
 
 		cc = &connectivity.Checker{}
+
+		// Check felix Wireguards have handshaked.
```

Comment:
handshook ? :smile:

Reply:
Or is it handshaken?

But seriously, I think "handshaked" is fine here.

Reply:
It sounds awful, the way it is I know, I think "Check felix nodes have performed Wireguard handshakes" is better do you? :)

Reply:
Yes, that last suggestion is nicer, thanks!